content/en/docs/tasks/administer-cluster/out-of-resource.md

```diff
@@ -140,6 +141,7 @@ The following node conditions are defined that correspond to the specified evict
 |-------------------|---------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------|
 | `MemoryPressure`  | `memory.available`                                                                    | Available memory on the node has satisfied an eviction threshold                                                             |
 | `DiskPressure`    | `nodefs.available`, `nodefs.inodesFree`, `imagefs.available`, or `imagefs.inodesFree` | Available disk space and inodes on either the node's root filesystem or image filesystem has satisfied an eviction threshold |
+| `PIDPressure`     | `pid.available`                                                                       | Available processes identifiers on the (Linux) node has fallen below an eviction threshold                                   |                                                         |
```

Comment:
"process identifiers" reads pretty strange to me. Maybe "PIDs" or "process IDs" is a better option.

Reply:
I don't think this is a blocking issue for merging this. "Process identifiers" just spells out what PIDs is short for.

Reply:
@celestehorgan Yes, it is not a blocker. However, I was arguing that when we are using established terms we should follow the tradition. In all operating system text books, academic papers or even tech overviews, we don't see "process identifiers". Similarly we don't say "user identifiers" or "group identifiers", we say "UID" or "user ID" instead. In this context, "PID" or "process ID" is a special integer. "Process identifier", on the other hand, sounds like a generalization of the term. It can be other things for identifying a process.

Reply:
We should write PID on this line, I think. It's OK to expand it though because PID can also mean proportional-integral-derivative, and Kubernetes uses _both_ meanings in different contexts.